OpenSim/Simulation/Model/Geometry.h

```diff
@@ -293,51 +293,17 @@ class OSIMSIMULATION_API Arrow : public Geometry
 
 
 /**
- * Utility class used to abstract analytic geometry. This will need to be 
- * revisited when wrapping is re-done to handle quadrants or analytic shapes
- * that were supported in earlier releases before 4.0. 
+ * Utility class used to abstract analytic geometry (surfaces of revolution)
  *
- * TODO: using start/end angle may be a better choice.
+ * TODO: using start/end angle to indicate partial surfaces.
  */
 class OSIMSIMULATION_API AnalyticGeometry : public Geometry
```

Comment:
Would appear that `AnalyticGeometry` is no longer necessary.

Reply:
There is nothing left.

Reply:
While the class is empty, it still serves a purpose to group types of geometry that have common features allowing uniform handling on the GUI/visualization side. 
All and only subtypes of AnalyticGeometry are built into the threejs library and performing operations on them requires different handling from other geometry primitives making the grouping useful.

Reply:
1.  What are the common defining features of `AnalyticGeometry` vs. `Geometry`?
2.  What makes an `Ellipsoid` "analytic" and a `Brick` not?
3. It seems this idea of `Analytic` was tied to using/drawing partial geometry. Is  that right?

Reply:
From abstraction point of view, they are surfaces of revolution. Because they are a well defined abstraction, the partial geometry rendering falls naturally in place, no surprise. 

From practical point of view, these types have built in implementation in threejs optimized for smooth rendering (compute derivatives, normals, tesselation etc.) so modifying their attributes after creation does not work, they have to be treated differently and recreated on edit. Not true for Mesh or Brick that can change attributes without recreation.

Hope this answers 1-3